ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -236,6 +242,11 @@ impl<'a> Assemble<'a> {
 
         let alloc_off = self.emit_prologue();
 
+        // The instruction offset after we've emitted the prologue (i.e. updated the stack
+        // pointer). We will later adjust this offset to also include one iteration of the trace
+        // (the preamble) so we can jump directly to the peeled loop.
+        let prologue_offset = self.asm.offset();
```

Comment:
Does RPython call this "preamble"? FWIW, I think I prefer "prologue" as the terminology.

Reply:
Yeah that's an RPython term. We can stick to prologue if you like. I'll fix it.

Reply:
Let's stick with prologue.

Reply:
[baf68c8](https://github.com/ykjit/yk/pull/1429/commits/baf68c8fb211101a86a6785dc45925de6fb989f7)

Reply:
Ah, I see why the change -- thanks!